tests/expand-manifest.jsonld

```diff
@@ -280,63 +352,18 @@
       "@id": "#tst27a",
       "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
       "name": "Invalid annotation node 9",
-      "purpose": "Node with @annotation that is an embedded node is invalid",
+      "purpose": "Node with @annotation that is a triple term is invalid",
       "input": "expand/st27a-in.jsonld",
       "expectErrorCode": "invalid annotation",
       "requires": "JSON-LD-star",
       "option": {"specVersion": "json-ld-star", "rdfstar": true}
-    }, {
-      "@id": "#tst28",
-      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
-      "name": "Embedded annotation node 1",
-      "purpose": "Node with @annotation property on embedded subject",
-      "input": "expand/st28-in.jsonld",
-      "expect": "expand/st28-out.jsonld",
-      "requires": "JSON-LD-star",
-      "option": {"specVersion": "json-ld-star", "rdfstar": true}
     }, {
       "@id": "#tst29",
-      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
       "name": "Embedded annotation node 1",
       "purpose": "Node with @annotation property on embedded object",
       "input": "expand/st29-in.jsonld",
-      "expect": "expand/st29-out.jsonld",
-      "requires": "JSON-LD-star",
-      "option": {"specVersion": "json-ld-star", "rdfstar": true}
-    }, {
-      "@id": "#tst30",
-      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
-      "name": "embedded node 11",
-      "purpose": "Embedded node with reverse relationship",
-      "input": "expand/st30-in.jsonld",
-      "expectErrorCode": "invalid embedded node",
-      "requires": "JSON-LD-star",
-      "option": {"specVersion": "json-ld-star", "rdfstar": true}
-    }, {
-      "@id": "#tst31",
-      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
-      "name": "embedded node 12",
-      "purpose": "Embedded node with expanded reverse relationship",
-      "input": "expand/st31-in.jsonld",
-      "expectErrorCode": "invalid embedded node",
-      "requires": "JSON-LD-star",
-      "option": {"specVersion": "json-ld-star", "rdfstar": true}
-    }, {
-      "@id": "#tst32",
-      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
-      "name": "embedded node 13",
-      "purpose": "Embedded node used as subject in reverse relationship",
-      "input": "expand/st32-in.jsonld",
-      "expect": "expand/st32-out.jsonld",
-      "requires": "JSON-LD-star",
-      "option": {"specVersion": "json-ld-star", "rdfstar": true}
-    }, {
-      "@id": "#tst33",
-      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
-      "name": "embedded node 14",
-      "purpose": "Embedded node used as object in reverse relationship",
-      "input": "expand/st33-in.jsonld",
-      "expect": "expand/st33-out.jsonld",
+      "expectErrorCode": "invalid triple term",
```

Comment:
why does this test become negative??
Seems to me that the result should be, in Turtle:
```
<ex:subj> <ex:value> <<( <ex:rei> <ex:prop> "value" )>> {| <ex:certainty> 8e-1 |}.
```
which is valid Turtle 1.2 ...

Reply:
The test input is the following:

```json
{
  "@id": "ex:subj",
  "ex:value": {
    "@triple": {
      "@id": "ex:rei",
      "ex:prop": "value"
    },
    "@annotation": {"ex:certainty": 0.8}
  }
}
```

This has `@annotation` in the same node as `@triple`. In my interpretation of JSON-LD-star, a triple term is an object containing only `@triple`. But, you're right that this should be allowed, but only for `@annotation`. The equivalent Turtle would be the following:

```turtle
<ex:subj> <ex:value> <<(<ex:rei> <ex:prop> "value")>> .
_:b0 rdf:reifies <<(<ex:subj> <ex:value> <<(<ex:rei> <ex:prop> "value")>>)>> .
_:b0 <ex:certainty> 0.8 .
```

Which is perfectly reasonable. I'll re-address this in another round.